src/test/java/com/cloudbees/hudson/plugins/folder/FolderTest.java

```diff
@@ -137,22 +138,27 @@
         JenkinsRule.WebClient wc = r.createWebClient();
 
         // "foo" should copy "child"
-        copyFromGUI(f, wc, "foo", "xyz");
+        copyViaHttp(f, wc, "foo", "xyz");
```

Comment:
AFAIU the test is testing that referring to a job via absolute path or relative '/foo' and `foo` works. The way to do that, via GUI or via and HTTP request, is IMHO irrelevant to the test. In fact the GUI at the end makes the same exact request, so this change is just avoiding an unnecesary and flaky step